src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -56,6 +59,65 @@ public class SchemaTypes : IEnumerable<IGraphType>
             [typeof(Uri)] = typeof(UriGraphType),
         });
 
+#if NET5_0_OR_GREATER
+        [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(GraphQLClrOutputTypeReference<int>))]
+        [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(GraphQLClrOutputTypeReference<long>))]
+        [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(GraphQLClrOutputTypeReference<BigInteger>))]
+        [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(GraphQLClrOutputTypeReference<double>))]
```

Comment:
I removed the calls to Preserve within this class by adding attributes to the metadata.  Perhaps this is more proper; but the code was a lot cleaner (and understandable) before.